etc/clean.py

```diff
@@ -1,18 +1,25 @@
+#!/usr/bin/env python
 import os
 
 
 def main():
     path = '.'
     count = 0
-    for name, folders, files in os.walk(path):
+    for name, folders, files in os.walk(path, topdown=False):
         for file in files:
             if file.endswith('.pyc') \
                or file.endswith('~') \
                or file.endswith('.pyo'):
                 count += 1
                 os.remove(os.path.join(name, file))
+        for folder in folders:
```

Comment:
do we need to do this in addition to deleting the `.pyc` files inside these directories?

Reply:
Yes. Python 3 saves `.pyc` files in `__pycache__` dirs.